pkg/downloader/chart_downloader.go

```diff
@@ -54,7 +54,7 @@ const (
 )
 
 // ErrNoOwnerRepo indicates that a given chart URL can't be found in any repos.
-var ErrNoOwnerRepo = errors.New("could not find a repo containing the given URL")
+var ErrNoOwnerRepo = errors.New("could not find a repo containing the given URL (try 'helm repo update')")
```

Comment:
This error is part of a package that can be used by part of the Helm CLI or part of other tools that import the library.  Suggesting to try `helm repo update` isn't going to work here. Something higher up should catch this error and add the helm command to help to it.

Reply:
@mattfarina There are 2 other occurrences of `(try 'helm repo update')` in this file (lines 284 and 289), which is why this messaging was used (in favor of consistency above all). I'd also argue that providing this messaging isn't misleading as it still cues the user to run the CLI command (even if they're not in the CLI). As a user, I wouldn't mind seeing this message in libraries/UIs/etc. If you still want a change, I'd rather remove the hint altogether if it gets this PR accepted faster. If that's also not an option, then I'll try to fit in some time to implement the requested change; let me know how you'd like to proceed.

Reply:
@mattfarina Just wanted to follow up on how you'd like to proceed ☝️ thanks!